packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -417,6 +420,20 @@ class CupertinoNavigationBar extends StatefulWidget implements ObstructingPrefer
   /// {@endtemplate}
   final bool transitionBetweenRoutes;
 
+  /// {@template flutter.cupertino.CupertinoNavigationBar.enableBackgroundFilterBlur}
+  /// Whether to have blur effect with non opaque color.
+  ///
+  /// If the [CupertinoNavigationBar] is not a child of a [CupertinoPageScaffold],
+  /// this has no effect.
+  ///
+  /// When [enableBackgroundFilterBlur] is set to false. The blur effect will be
```

Comment:
```suggestion
  /// When [enableBackgroundFilterBlur] is set to false, the blur effect will be
```

Reply:
Thanks for the approval, fixed the grammar issue as well.